### PR TITLE
Add file on continuous integration to the Quickstart

### PR DESCRIPTION
**Pull request by mmkerr**
Resolve issue #110 by adding file on continuous integration to the Quickstart (after 'uploading and validating config files').

### CODE REVIEW

File: docs/source/quickstart-hub-admin/continuous-integration.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+* available hubverse GitHub Actions
+
+
+## Installing continuous integration workflows with `hubCI`
+You can use the [`hubCI`](https://github.com/Infectious-Disease-Modeling-Hubs/hubCI) software package (R) to set up hubverse CI workflows. The development version can be installed with:
+
+``` r
+# install.packages("remotes")
+
+remotes::install_github("Infectious-Disease-Modeling-Hubs/hubValidations")
```
Comment (annakrystalli) [inline docs/source/quickstart-hub-admin/continuous-integration.md:16]:
```suggestion
remotes::install_github("Infectious-Disease-Modeling-Hubs/hubCI")
```

File: docs/source/quickstart-hub-admin/continuous-integration.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+The [`hubverse-actions`](https://github.com/Infectious-Disease-Modeling-Hubs/hubverse-actions) repository currently contains directories with templates related to the following workflows:
+
+* [`cache-hubval-deps`](https://github.com/Infectious-Disease-Modeling-Hubs/hubverse-actions/tree/main/cache-hubval-deps)
+* [`validate-submission`](https://github.com/Infectious-Disease-Modeling-Hubs/hubverse-actions/tree/main/validate-submission)
+* [`hubverse-aws-upload`](https://github.com/Infectious-Disease-Modeling-Hubs/hubverse-actions/tree/main/hubverse-aws-upload)
+
+More information on each of these workflows is provided in the section below. 
+
+If your hub is hosted on GitHub, you can download GitHub Actions using the command `use_hub_github_action()` with the name of the action in parentheses.
```
Comment (annakrystalli) [inline docs/source/quickstart-hub-admin/continuous-integration.md:28]: Again I think this needs to be clarified in that, if their hub is hosted on GitHub they can ***make use*** of our GitHub Actions. Anyone in an Rstudio project can download our actions

**Review by annakrystalli — CHANGES_REQUESTED** [review]
Overall, almost there! Just a couple of minor corrections requested.

**Review by nickreich — APPROVED** [review]
I am good with this as is, once @annakrystalli 's comments are addressed. thanks for addressing the comments I left in the Google Doc version of this!

File: docs/source/quickstart-hub-admin/continuous-integration.md (hunk 1 above)
Comment (mmkerr) [reply docs/source/quickstart-hub-admin/continuous-integration.md]: Done!

File: docs/source/quickstart-hub-admin/continuous-integration.md (hunk 2 above)
Comment (mmkerr) [reply docs/source/quickstart-hub-admin/continuous-integration.md]: @annakrystalli I have now highlighted this information in this section. In addition, the first paragraph also contains a sentence indicating that hubs must be hosted on GitHub to use GitHub Actions.

File: docs/source/quickstart-hub-admin/continuous-integration.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+# Setting up continuous integration via GitHub Actions
+
+Continuous integration (CI) is a practice that involves automating the way code and data are validated prior to being merged into a shared repository. This allows code and data to be built and tested when any changes are made, which can help users and developers identify and debug errors sooner. CI tasks are carried out via a *workflow*, an automated process with steps (i.e., *jobs*) that are run either sequentially or simultaneously. Workflows are triggered by an *event*, or a specific activity within a repository, such as a request to merge new code or data into a branch of a repository. Hubs that exist as GitHub repositories with hubverse-compliant data can use [GitHub Actions](https://github.com/features/actions) to perform a variety of CI workflows.
```
Comment (annakrystalli) [inline docs/source/quickstart-hub-admin/continuous-integration.md:3]:
```suggestion
Continuous integration (CI) is a practice that involves automating the way code and data are validated prior to being merged into a shared repository. This allows code and data to be built and tested when any changes are made, which can help users and developers identify and debug errors sooner. CI tasks are carried out via a *workflow*, an automated process with steps (i.e., *jobs*) that are run either sequentially or simultaneously. Workflows are triggered by an *event*, or a specific activity within a repository, such as a request to merge new code or data into a branch of a repository. Hubs that are hosted on GitHub with hubverse-compliant structure can use [GitHub Actions](https://github.com/features/actions), and specifically the hubverse GitHub Action templates we've developed to perform a variety of CI workflows. 
```

**Review by annakrystalli — APPROVED** [review]
All good from me

**Pull request merged**